fbw-common/src/wasm/systems/systems/src/hydraulic/nose_steering.rs

```diff
@@ -116,9 +115,14 @@ impl SteeringActuator {
 
     const REFERENCE_PRESS_FOR_NOMINAL_SPEED_PSI: f64 = 2000.;
 
-    const MAX_SPEED_FILTER_TIMECONST: Duration = Duration::from_millis(100);
     const CURRENT_SPEED_FILTER_TIMECONST: Duration = Duration::from_millis(150);
 
+    // Adjusts how the steering slows down with position error
+    //   Formula is speed_coefficient = POSITION_ERROR_TO_MAX_SPEED_GAIN * position_error^2
+    //   Then max speed will be max_speed = nominal_speed * speed_coefficient
+    //   Note this is open loop: it will overshoot or undershoot depending on this factor
```

Comment:
```suggestion
    // Formula is speed_coefficient = POSITION_ERROR_TO_MAX_SPEED_GAIN * position_error^2
    // Then max speed will be max_speed = nominal_speed * speed_coefficient
    // Note this is open loop: it will overshoot or undershoot depending on this factor
```